Client/Frontend/Reader/Reader.css

```diff
@@ -0,0 +1,588 @@
+  bottom: 0px;
+  left: 0px;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  background-color: #EBEBF0;
+  -moz-user-select: none;
+}
+
+.toolbar-hidden {
+  transition-property: visibility, opacity;
+  transition-duration: 0.7s;
+  visibility: hidden;
+  opacity: 0.0;
+}
+
+.toolbar > * {
+  float: right;
+  width: 33%;
+}
+
+.button {
+  color: white;
+  display: block;
+  background-position: center;
+  background-size: 30px 24px;
+  background-repeat: no-repeat;
+}
+
+.dropdown {
+  text-align: center;
+  display: inline-block;
+  list-style: none;
+  margin: 0px;
+  padding: 0px;
+}
+
+.dropdown li {
+  margin: 0px;
+  padding: 0px;
+}
+
+.dropdown-toggle {
+  margin: 0px;
+  padding: 0px;
+}
+
+.dropdown-popup {
+  text-align: start;
+  position: absolute;
+  left: 0px;
+  z-index: 1000;
+  float: left;
+  background: #EBEBF0;
+  margin-top: 12px;
+  margin-bottom: 10px;
+  padding-top: 4px;
+  padding-bottom: 8px;
+  font-size: 14px;
+  box-shadow: 0px -1px 12px #333;
+  border-radius: 3px;
+  visibility: hidden;
+}
+
+.dropdown-popup > hr {
+  width: 100%;
+  height: 0px;
+  border: 0px;
+  border-top: 1px solid #B5B5B5;
+  margin: 0;
+}
+
+.open > .dropdown-popup {
+  margin-top: 0px;
+  margin-bottom: 6px;
+  bottom: 100%;
+  visibility: visible;
+}
+
+.dropdown-arrow {
+  position: absolute;
+  width: 40px;
+  height: 18px;
+  bottom: -18px;
+  background-image: url('chrome://browser/skin/images/reader-dropdown-arrow-mdpi.png');
+  background-size: 40px 18px;
+  background-position: center;
+  display: block;
+}
+
+#font-type-buttons,
+.segmented-button {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  padding: 10px 5px;
+  white-space: nowrap;
+}
+
+#font-type-buttons > li,
+.segmented-button > li {
+  width: 50px; /* combined with flex, this acts as a minimum width */
+  flex: 1 0 auto;
+  text-align: center;
+  line-height: 20px;
+}
+
+#font-type-buttons > li {
+  padding: 10px 0;
+}
+
+.segmented-button > li {
+  border-left: 1px solid #B5B5B5;
+}
+
+.segmented-button > li:first-child {
+  border-left: 0px;
+}
+
+#font-type-buttons > li > a,
+.segmented-button > li > a {
+  vertical-align: middle;
+  text-decoration: none;
+  color: black;
+}
+
+#font-type-buttons > li > a {
+  display: inline-block;
+  font-size: 48px;
+  line-height: 50px;
+  margin-bottom: 5px;
+  border-bottom: 3px solid transparent;
+}
+
+.segmented-button > li > a {
+  display: block;
+  padding: 5px 0;
+  font-family: "Clear Sans",sans-serif;
+  font-weight: lighter;
+}
+
+#font-type-buttons > li > a:active,
+#font-type-buttons > li.selected > a {
+  border-color: #ff9400;
+}
+
+.segmented-button > li > a:active,
+.segmented-button > li.selected > a {
+  font-weight: bold;
+}
+
+#font-type-buttons > li > .sans-serif {
+  font-weight: lighter;
+}
+
+#font-type-buttons > li > div {
+  color: #666;
+  font-size: 12px;
+}
+
+.toggle-button.on {
+  background-image: url('chrome://browser/skin/images/reader-toggle-on-icon-mdpi.png');
```

Comment:
@st3fan I started trying to split out the content styles into a shared file while working on bug 1117258, but after talking with UX, I decided to hold off on that, since there are still some differences that they want between Android/desktop. However, in the long run, I would ideally like to have something shared for the article text, since those styles should be similar.